health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/DeliveryDetailsSummary.js

```
@@ -85,45 +86,93 @@ function reverseDeliveryRemap(data, t) {
+                name: variant.name
+            }));
+
+            const condition = criteria.condition;
+            let conditionParts = condition.split("and").map(part => part.trim()); // Split by 'and' and trim spaces
+            let rules = [];
+
+            // Iterate over each part of the condition if split by 'and'
+            conditionParts.forEach((part) => {
+                const parts = part.split(' ').filter(Boolean); // Split by space and remove empty strings
+
+                let attributes = [];
```

Biome

[error] This let declares a variable that is only assigned once.

'attributes' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)